packages/client/components/GitHubScopingSelectAllIssues.tsx

```diff
@@ -86,7 +86,12 @@ const GitHubScopingSelectAllIssues = (props: Props) => {
       )
       return issue?.title ?? 'Unknown Story'
     })
-    UpdatePokerScopeMutation(atmosphere, variables, {onError, onCompleted, contents})
+    UpdatePokerScopeMutation(atmosphere, variables, {
+      onError,
+      onCompleted,
+      contents,
+      selectedAll: true
```

Comment:
To track whether a user has selected all issues, we could try to calculate this in the mutation, but it would be tricky. We'd need to query all of the issues from each integration using the same first arg. We'd then need to check which issues had been previously selected and what the `Threshold.MAX_POKER_STORIES` is. Then, if the number of issues added to the mutation is equal to that calculation, we'd know the selectAll button had been selected.

While I don't love passing an extra variable to the mutation, I think it's less error-prone and easier to maintain than the calculation solution mentioned above.